app/views/browse/pod.html

```diff
@@ -122,73 +122,84 @@
+                                    <span
+                                      ng-if="term.isUsed && (term.containerState === 'running')"
+                                      ng-class="{'text-muted' : (term.status === 'disconnected')}">
+                                        {{term.status}}
+                                    </span>
+                                    <span
+                                      ng-if="term.containerState !== 'running'"
+                                      ng-class="{'text-muted' : !term.isUsed}">
+                                        {{term.containerState}}
+                                    </span>
+                                  </div>
+                                </ui-select-choices>
+                              </ui-select>
+                            </div>
                           </div>
-                        </div>
 
-                        <div class="container-terminal-wrapper">
-                          <div class="row" ng-repeat="term in containerTerminals">
-                            <div class="column">
-                              <kubernetes-container-terminal
-                                prevent="!terminalTabWasSelected"
-                                ng-if="term.isUsed"
-                                ng-show="term.isVisible"
-                                pod="pod"
-                                container="term.containerName"
-                                status="term.status"
-                                rows="terminalRows"
-                                cols="terminalCols"
-                                autofocus="true"
-                                >
-                              </kubernetes-container-terminal>
+                          <div class="container-terminal-wrapper">
+                            <div class="row" ng-repeat="term in containerTerminals">
+                              <div class="column">
+                                <kubernetes-container-terminal
+                                  prevent="!terminalTabWasSelected"
+                                  ng-if="term.isUsed"
+                                  ng-show="term.isVisible"
+                                  pod="pod"
+                                  container="term.containerName"
+                                  status="term.status"
+                                  rows="terminalRows"
+                                  cols="terminalCols"
+                                  autofocus="true"
+                                  >
+                                </kubernetes-container-terminal>
+                              </div>
                             </div>
                           </div>
                         </div>
                       </div>
-    	            </uib-tab>
+
+                      <div ng-if="containersRunning(pod.status.containerStatuses) <= 0" class="blank-slate-pf table-log-pods" >
```

Comment:
Can you switch to use the `empty-state-message` class we use elsewhere until we switch everything to `blank-slate-pf`?

Here's an example we use for the debug terminal:

https://github.com/openshift/origin-web-console/blob/e6249b0435b9fea2401cbc7297de16d1cdcbe172/app/views/modals/debug-terminal.html#L11-L28